deploy/chart/values.yaml

```diff
@@ -93,11 +93,33 @@ configmap:
   # specify the custom script for setup and teardown
   setup: |-
     #!/bin/sh
-    path=$1
-    mkdir -m 0777 -p ${path}
+    type=$1
```

Comment:
We should use e.g. `--type` kind of parameter to separate them. Otherwise, it's easy to mix up the parameters.

In fact, we can just treat it as a boolean `--block`.